tests/test_models/test_forward/test_sot_forward.py

```diff
@@ -12,7 +12,7 @@
     'sot/siamese_rpn/siamese_rpn_r50_20e_lasot.py',
```

Comment:
follow [this](https://github.com/open-mmlab/mmtracking/blob/master/tests/test_models/test_track_heads/test_siamese_rpn_head.py) to add ut for stark_hea